gethelpcmd-*, saracmd-*: add page (#21946)
tldr-pages/tldr: pages/windows/gethelpcmd-expertexperienceadmintask.md

# GetHelpCmd ExpertExperienceAdminTask

> Get detailed installation and configuration information on legacy Microsoft Outlook with Microsoft Office and Windows.
> Part of `GetHelpCmd.exe`, formerly `SaRAcmd.exe` (Microsoft Support and Recovery Assistant).
> Note: This tool has been deprecated and will not work in the new Outlook application.
> More information: <https://learn.microsoft.com/troubleshoot/microsoft-365/admin/miscellaneous/get-help-outlook-scan>.

- Get detailed information and accept this command's End-User License Agreement (EULA):

`GetHelpCmd.exe -S ExpertExperienceAdminTask -AcceptEula`

> - Get detailed information and save logs to a file:

Reviewer: ```suggestion
- Get detailed information and save the logs to a specific directory:
```

> `GetHelpCmd.exe -S ExpertExperienceAdminTask -AcceptEula -LogFolder {{path\to\file}}`

Reviewer: ```suggestion
`GetHelpCmd.exe -S ExpertExperienceAdminTask -AcceptEula -LogFolder {{path\to\directory}}`
```

> - Get detailed information and hide the program progress:

Reviewer: ```suggestion
- Get detailed information without showing the progress:
```
